test/blackbox-tests/test-cases/melange/virtual_lib.t/run.t

```diff
@@ -1,5 +1,12 @@
-Virtual library with a single module are not yet supported for melange libs
+Virtual library with a single module are supported by melange libs
 
   $ dune build output/c.js
   $ node _build/default/output/c.js
   hello from melange
+
+Implementation modules in the virtual library are supported
+
+  $ touch vlib/shared.ml
+  $ dune build output/c.js
```

Comment:
was this returning an error before? can you `ls _build/default/output` to check that `shared.js` ends up there?

Reply:
Yes, see https://github.com/ocaml/dune/pull/6495/commits/72769c8471685b972c7bb0128611f7ddf0b30831.

> can you ls _build/default/output to check that shared.js ends up there?

Even more, I am thinking about updating the test to make it something realistic: have exe / melange implementations, intra-deps in the vlib, in the impl lib, etc. I think some cases are still broken, due to `obj_dir` of vlib not being taken into account.

Reply:
added more complete test in https://github.com/ocaml/dune/pull/6495/commits/f808e3bc50a854c22f057c007e56bc2561bf797f.